rpm/centos-8/Dockerfile

```diff
@@ -18,27 +18,23 @@ ARG SUITE
 ENV DISTRO=${DISTRO}
 ENV SUITE=${SUITE}
 
-# In aarch64 (arm64) images, the altarch repo is specified as repository, but
-# failing, so replace the URL.
-RUN if [ -f /etc/yum.repos.d/CentOS-Stream-Sources.repo ]; then sed -i 's/altarch/centos/g' /etc/yum.repos.d/CentOS-Stream-Sources.repo; fi
-
```

Comment:
This one was an odd one; I know we added it at some point, then we removed it (because it appeared it didn't have the `altarch` anymore); https://github.com/docker/docker-ce-packaging/pull/416#discussion_r367891915

But later, we had to add it back, because for some reason, our pipeline started failing again on this; see https://github.com/docker/docker-ce-packaging/pull/446

Could it be these can be set dynamically, depending on what mirror is hit?